ocfweb/context_processors.py

```diff
@@ -14,13 +14,19 @@
 
 
 def get_base_css_classes(request):
-    if request.resolver_match.url_name:
+    if not request.resolver_match:
+        # Need this for the 404 page
+        yield ''
+    elif request.resolver_match.url_name:
         page_class = 'page-' + request.resolver_match.url_name
         yield page_class
 
         for arg in request.resolver_match.args:
             page_class += '-' + re.sub(r'[^a-zA-Z_\-]', '-', arg)
             yield page_class
+    else:
```

Comment:
do what @jvperrin suggested earlier, which is also what we did when we looked at this together a couple weeks ago, rather than ending up raising the rather under-descriptive `Exception`